roles/dns/manage-dns-records/tasks/route53/process-one-record.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: "Apply workaround fix for Route53 TXT records"
+  set_fact:
+    record_value: "{{ (item.type == 'TXT') | ternary('\"' + item.value + '\"', item.value) }}"
+  when:
+    - item.value is defined 
+
+- route53:
+    zone: "{{ dns.1.dns_domain }}"
+    record: "{{ item.record }}"
+    value: '{{ record_value | default(omit) }}'
+    type: "{{ item.type }}"
+    ttl: "{{ item.ttl | default(omit) }}"
+    overwrite: "{{ item.overwrite | default(omit) }}"
+    private_zone: "{{ item.private_zone | default(omit) }}"
+    region: "{{ item.region | default(omit) }}"
+    vpc_id: "{{ item.vpc_id | default(omit) }}"
+    hosted_zone_id: "{{ item.hosted_zone_id | default(omit) }}"
```

Comment:
Since `private_zone`, `region`, `vpc_id` & `hosted_zone_id` will be the same for all records in a zone, would it make sense to put them as `dns.1.route53.private_zone`, `dns.1.route53.region`, `dns.1.route53.vpc_id` & `dns.1.route53.hosted_zone_id`?
This way they don't have to be set for each individual record.

Reply:
Make sense. It aligns with the existing sample inventory. Will also write an issue to address additional parameters.